read_test.go

```diff
@@ -364,14 +366,133 @@ func TestReadNativeFrames(t *testing.T) {
 			expectedPixelData: nil,
 			expectedError:     ErrorUnsupportedBitsAllocated,
 		},
+		{
+			Name: "3x3, 3 frames, 1 samples/pixel",
```

Comment:
```suggestion
			Name: "3x3, 3 frames, 1 samples/pixel, data bytes with padded 0",
```
May be good to include the padded 0 part in the test case name.